api/v3/CiviOffice/Convert.php

```diff
@@ -29,37 +29,37 @@ function _civicrm_api3_civi_office_convert_spec(&$spec)
         'name'         => 'document_uri',
         'api.required' => 1,
         'type' => CRM_Utils_Type::T_STRING,
-        'title'        => E::ts('Document URI. E.g.: "local::common/vorlage_kontakte_und_zuwendungen.docx"'),
+        'title'        => E::ts('Document URI, e.g. "local::common/template_contacts_and_contributions.docx"'),
         'description'  => E::ts('URI of document.'),
     ];
     $spec['entity_ids'] = [
         'name'         => 'entity_ids',
         'api.required' => 1,
         'type' => CRM_Utils_Type::T_INT,
-        'title'        => E::ts('Array of entity ids. E.g.: "[362, 614]"'),
+        'title'        => E::ts('Array of entity ids, e.g. "[362, 614]"'),
```

Comment:
Let's use some more example-y IDs, e.g. `123, 456`. Also, the example should not be part of the title, but the description.

Reply:
Changed.